js/toolbar.js

```diff
@@ -298,7 +300,7 @@ class Toolbar {
                 }
             } else {
                 if (elem !== undefined && elem !== null) {
-                    elem.setAttribute("data-tooltip", trans);
+                    elem.setAttribute("data-tooltip", trans);           
```

Comment:
please remove the trailing spaces

Reply:
Can you please explain why this fixes the problem?

Reply:
Actually there was one recent PR of toggle full screen which caused this issue,and in that PR "FullScreen"  trans in "strings_"  was not set causing a subsequent change of tooltips of all the other icons while looping through all the strings